rioting-daisies/BangazonSite
# Jh images

# Description

Added ability to upload images in the sell item form.  Added being able to view image in product details and index view. Images show up with items are searched and pictures are 75px by 75px.



Fixes Issue https://github.com/rioting-daisies/BangazonSite/issues/18 
https://github.com/rioting-daisies/BangazonSite/issues/19
## Type of change

Please delete options that are not relevant.



# Testing Instructions

- git fetch --all
- git checkout jh-images
- save an image you want for the item you want to sell on your local computer
- click sell something and fill out the form along with uploading an image
- look at index and details pages to make sure image you uploaded is there
- make sure picture is 75px by 75px
-search items that have pictures and see if image is still there


# Checklist:
